crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/with.py

```diff
@@ -86,10 +86,6 @@
     )
 ): ...
 
-with (a # trailing same line comment
-    # trailing own line comment
-    ) as b: ...
-
```

Comment:
What's the reason for removing this test case?

Reply:
It's a duplicate, sorry should have added a PR comment